app/kafka/manager/utils/AdminUtils.scala

```diff
@@ -114,6 +114,82 @@ class AdminUtils(version: KafkaVersion) {
     createOrUpdateTopicPartitionAssignmentPathInZK(curator, topic, replicaAssignment, topicConfig)
   }
 
+  def deleteLogkafka(curator: CuratorFramework, 
```

Comment:
Let's move these to LogKafkaAdminUtils as AdminUtils is meant to replicate code from Kafka's AdminUtils class.